client/src/api.ts

```diff
@@ -98,27 +98,34 @@ export interface PaginatedNotifications {
 }
 
 export type QrCode = {
+  beneficiary: string;
+  claimed: boolean;
+  claimed_date: string;
+  created_date: string;
+  event_id: number;
   id: number;
+  is_active: boolean;
+  numeric_id: number;
   qr_hash: string;
-  claimed: boolean;
+  qr_roll_id: number;
   tx_hash: string;
-  event_id: number;
   event: PoapEvent;
 };
 
 export type PaginatedQrCodes = {
   limit: number;
   offset: number;
   total: number;
-  codes: QrCode[];
+  qr_claims: QrCode[];
 };
 
 export type ENSQueryResult = { valid: false } | { valid: true; address: string };
 
 export type AddressQueryResult = { valid: false } | { valid: true; ens: string };
 
-let API_BASE = 'https://api.poap.xyz';
-// let API_BASE = 'http://10.0.0.146:8080';
+// let API_BASE = 'https://api.poap.xyz';
+let API_BASE = 'http://10.0.0.146:8080';
+// let API_BASE = 'https://development-dot-poapapp.appspot.com';
```

Comment:
This `API_BASE` are getting a little out of hand, can we have something like:

```typescript
const API_URLS = {
  prod: 'https://api.poap.xyz',
  test: 'https://development-dot-poapapp.appspot.com',
  local: 'http://10.0.0.146:8080',
}

const API_BASE = process.env.NODE_ENV === 'development' ? API_URLS.local : API_URLS.prod;
```

So it's easier to switch between `test` and `local`, let me know what you think.